source/core/src/main/com/csse3200/game/components/inventory/ToolbarDisplay.java

```diff
@@ -107,14 +107,70 @@ private void addActors() {
         table.defaults().size(64, 64);
 
         for (int i = 0; i < 10; i++) {
+            //Set the indexes for the toolbar
+            int idx = i + 1;
+            if (idx == 10) {
+                idx = 0;
+            }
+
+            //Create the label for the item slot
+            Label label = new Label(String.valueOf(idx) + " ", skin); //please please please work
+            label.setColor(Color.DARK_GRAY);
+            label.setAlignment(Align.topLeft);
+
+            //Create the itemslot
+            ItemSlot item = new ItemSlot(false);
```

Comment:
This does not achieve the desired objective of the ticket.

What we _need_ to do is to check if a slot is currently active and then modify the item slot accordingly. At this point it doesn't do this. You could do this a number of ways, one of which is to follow how Nimesh updated the player inventory in toolbar, or to use custom events. Let me know if you need help with this.

Reply:
Okay I'll try to update it before the end of the sprint.

Reply:
> Okay I'll try to update it before the end of the sprint.

Let me know if this isn't feasible. I've got some time to do this if you don't tonight.